src/status_im2/common/bottom_sheet/view.cljs

```diff
@@ -205,22 +207,26 @@
                         {:transform [{:translateY translate-y}]}
                         {:width  window-width
                          :height window-height})}
-               [rn/view {:style (styles/background)}
-                [rn/keyboard-avoiding-view
-                 {:behaviour (if platform/ios? :padding :height)
-                  :style     {:flex 1}}
-                 [rn/view
-                  {:style     {:position       :absolute
-                               :left           0
-                               :right          0
-                               :top            0
-                               :padding-top    styles/border-radius
-                               :padding-bottom (:bottom insets)}
-                   :on-layout (when-not (and
-                                         (some? @content-height)
-                                         (> @content-height 0))
-                                on-content-layout)}
-                  children]]
+               [rn/view {:style (styles/container)}
+                (when selected
+                  [rn/view {:style styles/selected-background}
+                   [selected]])
+                [rn/view {:style (styles/background)}
+                 [rn/keyboard-avoiding-view
+                  {:behaviour (if platform/ios? :padding :height)
+                   :style     {:flex 1}}
+                  [rn/view
+                   {:style     {:position       :absolute
+                                :left           0
+                                :right          0
+                                :top            0
+                                :padding-top    styles/border-radius
+                                :padding-bottom (:bottom insets)}
+                    :on-layout (when-not (and
```

Comment:
For your information:

ClojureScript doesn't have the same issues as Javascript, where some unexpected things are falsey, like an empty string or the number zero. So in ClojureScript (as well as many other languages) it's safe and expected to just check for existence:

```clojure
(when-not (and @content-height (pos? @content-height))
  ...)
```